src/main/java/ecdar/abstractions/SystemRoot.java

```diff
@@ -33,12 +33,12 @@ public DoubleProperty getXProperty() {
 
     @Override
     public ObservableValue<? extends Number> getEdgeX() {
-        return getXProperty().add(3 * Grid.GRID_SIZE);
+        return getXProperty().add(Ecdar.CANVAS_PADDING * 3);
     }
 
     @Override
     public ObservableValue<? extends Number> getEdgeY() {
-        return new SimpleDoubleProperty(Grid.TOOL_BAR_HEIGHT + 2 * Grid.GRID_SIZE - 4d);
+        return new SimpleDoubleProperty(Ecdar.CANVAS_PADDING * 4 - 4d);

```

Comment:
Are we sure we don't want to keep variables for the toolbar height (`TOOL_BAR_HEIGHT`) and corner size (`CORNER_SIZE`). It seems like they were used often before and now they are replaced by magical constants, are they no longer useful without the grid?